app/models/content/concerns/has_children.rb

```diff
@@ -20,17 +20,19 @@ def present?
     true
   end
 
-  def reflow_contents
-    reflow_tree tree, ordinals
+  def reflow_contents(child)
+    @child = child
+    reflow_tree(tree, ordinals)
   end
 
-  def reflow_tree tree, prefix = []
+  def reflow_tree(tree, prefix = [])
     ord = 1
     tree.each do |element|
       if element.is_a? Enumerable
-        reflow_tree element, ord == 1 ? prefix : prefix + [ord-1]
+        element_prefix = ord == 1 ? prefix : prefix + [ord-1]
```

Comment:
As long as you’re in here, let’s get rid of the ternary and replace it with an `if`🙂. It took me a while to parse it.

Reply:
Also, are you sure this will do what you want? Maybe I’m missing something, but if you always set ord to 1 at the start of the method, this conditional will always return true.

Reply:
No it's looping through the tree after setting ord to 1...  so it's not always 1